packages/dapp/src/components/Navbar/NavbarTabsContainer.tsx

```diff
@@ -17,17 +18,21 @@ const NavbarTabsContainer = () => {
   const theme = useTheme();
   const { t: translate } = useTranslation();
   const i18Path = 'navbar.';
-  const settings = useSettings();
+  const activeTab = useSettingsStore((state: any) => state.activeTab);
+  const onChangeTab = useSettingsStore(
+    (state: SettingsContextProps) => state.onChangeTab,
+  );
```

Comment:
I guess we don't need to explicitly define state type here (SettingsContextProps or any), as it should be derived from function types. The same applies to other places.

Reply:
That´s right Eugene, thanks to that find I could refactor and remvoe a bunch of stuff. Your example code in the widget was a big help on that one.